docs/book/src/topics/crd-changes/v1alpha6-to-v1alpha7.md

```diff
@@ -258,6 +258,10 @@ known to cause some issues in some environments.
 
 ### `OpenStackCluster`
 
+#### ⚠️ Change to floatingIP
+
+The `floatingIP` field has moved from the `OpenStackMachineSpec` to the `OpenStackClusterSpec` into the `Bastion` spec.
```

Comment:
Could you change this so it explains the change in yaml instead of in code? For a user that is not familiar with the go code this will be hard to grasp

Reply:
yes I'll add it.